README.md

```diff
@@ -142,6 +144,84 @@ cisco_interface_ospf {"Ethernet1/2 Sample":
 }
 ~~~
 
+## Platform Support
+
+### <a name="provider-platform-support">Provider Support Across Platforms</a>
+
+The following table indicates which providers are supported on each platform. As platforms are added to the support list they may indicate ' `Unsupported`' for some providers that have not completed the test validation process at the time of this release. Some providers will show caveats for a platform if there are limitations on usage, such as with unsupported properties or hardware limitations.
+
+##### Cisco Providers
+
+| ✅ = Supported <br> ❌ = Unsupported | N9k | N30xx | N31xx | N56xx | N6k | N7k | Caveats |
+|:---|:---:|:-----:|:-----:|:-----:|:---:|:---:|:---:|
+| [cisco_aaa_authentication_login](#type-cisco_aaa_authentication_login) | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| [cisco_aaa_authorization_cfg_svc](#type-cisco_aaa_authorization_cfg_svc) | ✅ | ✅ | ✅ | ❌ | ✅ | ❌ |
```

Comment:
While I realize that we do have passing results for beaker on n6k for 1.1.0+ providers, I am not sure I am comfortable with calling them `Supported` without the N(5|7)K results.  Given that N5K runs the same software and N7K also supports OAC, we might get undesired questions about why the other platforms are not also supported?